src/pseudopeople/schema_entities.py

```diff
@@ -147,7 +147,7 @@ class __Columns(NamedTuple):
         "itin",
         (
             NOISE_TYPES.leave_blank,
-            # NOISE_TYPES.copy_from_within_household,
+            NOISE_TYPES.copy_from_household_member,
```

Comment:
Isn't there still ITIN implementation needed to be done?

Reply:
This should not be changed. The 1040 would need to be updated and we would need to implement copying ITIN in PRL.